src/configuration.js

```diff
@@ -39,15 +38,15 @@ export class Configuration {
       return this.schema;
     }
 
-    var schema;
-
     if (this.options.stdin) {
       this.schema = getSchemaFromFileDescriptor(this.stdinFd);
       this.sourceMap = new SourceMap({ stdin: this.schema });
     } else if (this.options.schemaPaths) {
-      var expandedPaths = expandPaths(this.options.schemaPaths);
-      var segments = getSchemaSegmentsFromFiles(expandedPaths);
-
+      const expandedPaths = expandPaths(this.options.schemaPaths);
+      const segments = getSchemaSegmentsFromFiles(expandedPaths);
+      if (Object.keys(segments).length === 0) {
+        return this.schema;
```

Comment:
Should this be `return null;`? I think that's essentially what will happen, but I thought returning `this.schema` was a bit confusing.